test/plugin/scenarios/thrift-scenario/support-version.list

```diff
@@ -17,3 +17,4 @@
 0.12.0
 0.11.0
 0.10.0
+0.9.3
```

Comment:
From the test log, https://pipelines.actions.githubusercontent.com/serviceHosts/c62bd4a6-a619-4835-aa78-99798d940fc7/_apis/pipelines/1/runs/13668/signedlogcontent/18?urlExpires=2023-03-02T12%3A54%3A46.4948869Z&urlSigningMethod=HMACV1&urlSignature=nKo%2FKZeAMBo7y2mS0i%2Fl80EobjyjEsN7ami57gKjrao%3D

This test scenario can't be compiled in 0.9.3. Please check locally.

Reply:
@zhaoxiaojie0415 Is there any update about this?

Reply:
> @zhaoxiaojie0415 Is there any update about this?

In thrift 0.9.3, it generate the interface with the param Generic erase which leads to test case fail. 
Is it appropriate to write two versions of test cases?  （Actually 0.9.3 and 0.10.0 use same plugin code, but need to use different test cases）

Reply:
Sorry, I encouter a maven wrapper problem in local test env which can not package test/plugin/pom.xml and then can not run test cases locally. It would take me more time to solve it.

Reply:
Generally, as 0.9.3 is a old version, we don't have to put that in the test list, if others can pass.

> I encouter a maven wrapper problem in local test env

You may need a proxy, sometimes the network doesn't work.

Reply:
> Generally, as 0.9.3 is a old version, we don't have to put that in the test list, if others can pass.
> 
> > I encouter a maven wrapper problem in local test env
> 
> You may need a proxy, sometimes the network doesn't work.

Thanks. I remove the 0.9.3 in the latest submit. Between 0.9.3 and 0.10.0, there has some difference for async method(for example, the interface of 0.9.3 generic type erasure， while 0.10.0 not), which leads to the test case fail.